build/build.xml

```diff
@@ -413,9 +413,6 @@
 		<build-addon name="tlsdebug" />
 		<build-addon name="viewstate" />
 		<build-addon name="wappalyzer" />
-		<!-- Need to download the drivers before building the webdriver add-ons -->
-		<antcall target="clean-webdrivers"/>
```

Comment:
This means that the WebDrivers are no longer re-downloaded, that's on purpose? (I'm fine with this behaviour.)